tests/test_connector.py

```diff
@@ -2179,10 +2179,23 @@ def test_not_expired_ttl(self) -> None:
         dns_cache_table.add("localhost", ["127.0.0.1"])
         assert not dns_cache_table.expired("localhost")
 
-    async def test_expired_ttl(self, loop: Any) -> None:
+    async def test_expired_ttl(self, loop: Any, monkeypatch: Any) -> None:
         dns_cache_table = _DNSCacheTable(ttl=0.01)
+        monkeypatch.setattr("aiohttp.connector.monotonic", lambda: 1)
+        dns_cache_table.add("localhost", ["127.0.0.1"])
+        monkeypatch.setattr("aiohttp.connector.monotonic", lambda: 1.02)
+        assert dns_cache_table.expired("localhost")
```

Comment:
```suggestion
        dns_cache_table = _DNSCacheTable(ttl=1)
        monkeypatch.setattr("aiohttp.connector.monotonic", lambda: 1)
        dns_cache_table.add("localhost", ["127.0.0.1"])
        monkeypatch.setattr("aiohttp.connector.monotonic", lambda: 3)
        assert dns_cache_table.expired("localhost")
```
Think we can just do simple numbers here, given that we are patching the clock.

Reply:
Makes sense, added "nicer" numbers.